ffi/SDL2_0_h.lua

```diff
@@ -792,6 +792,7 @@ void SDL_Delay(Uint32) __attribute__((visibility("default")));
 int SDL_GetCurrentDisplayMode(int, SDL_DisplayMode *) __attribute__((visibility("default")));
 void SDL_EnableScreenSaver(void) __attribute__((visibility("default")));
 SDL_Window *SDL_CreateWindow(const char *, int, int, int, int, Uint32) __attribute__((visibility("default")));
+void SDL_GetWindowSize(SDL_Window *, int *, int *) __attribute__((visibility("default")));
```

Comment:
Please also add this to https://github.com/koreader/koreader-base/blob/eef692467039d445966f3e303f0c7b63389be1f4/ffi-cdecl/SDL2_0_decl.c